app/views/users/show.html.erb

```diff
@@ -1,3 +1,16 @@
-<h1>User Details</h1>
-<p><%= @placeholder_text %></p>
-<p>Find me in app/views/users/show.html.erb</p>
+<%= render partial: 'users/user', locals: { user: @user } %>
+<section id="user-details">
+  <div>
+    <h3>Bio</h3>
+    <p><%= @user.bio %></p>
+  </div>
+  <hr>
+  <section id="posts-container">
+    <% @user.posts.each do |post| %>
+      <%= render partial: 'posts/post', locals: { post: post } %>
+    <% end %>
+  </section>
```

Comment:
- [ ] Great job displaying the posts but I think it would be great if you just displayed the most recent posts as shown in the wireframe and to do that you can Use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).
so please update this as well to just show in the users / show page the most 3 recent posts 🎉🔥



- your view
![image](https://github.com/richie1988/blog-app/assets/58553711/7d0ec004-854f-48f0-9768-228c38b31a37)